src/hotspot/share/jfr/metadata/metadata.xml

```diff
@@ -706,6 +706,19 @@
     <Field type="OldObjectGcRoot" name="root" label="GC Root" />
   </Event>
 
+  <Event name="NativeMemoryUsage" category="Java Virtual Machine, Memory" label="Native Memory Usage Per Type"
+    description="Native memory usage for a given memory type in the JVM" period="everyChunk">
+    <Field type="string" name="type" label="Memory Type" description="Type used for the native memory allocation" />
```

Comment:
Suggest turning this type "string" into a JFR-constant type, like "JfrNMTType" to only send id's, instead of the entire strings. Can be done in a follow-up change.

Reply:
I prefer to do this at a later point, created [JDK-8298278](https://bugs.openjdk.org/browse/JDK-8298278) to track this.